src/utils/update/trust.model.fetch.ts

```diff
@@ -71,13 +71,22 @@ const retrieveTrusts = async (req: Request, appData: ApplicationData) => {
 
 export const mapTrustData = (trustData: TrustData, appData: ApplicationData) => {
   const dateOfBirth = mapInputDate(trustData.creationDate);
+
+  let stillInvolved = trustData.trustStillInvolvedInOverseasEntityIndicator === "1" ? "Yes" : "No";
+
+  // If 'trustStillInvolvedInOverseasEntityIndicator' isn't set (could be null or undefined), need to set an empty string
+  if (!trustData.trustStillInvolvedInOverseasEntityIndicator) {
```

Comment:
would it be better to check if trustStillInvolvedInOverseasEntityIndicator is not the expected values ("1", "0") instead of ! as if the trustStillInvolvedInOverseasEntityIndicator is set to "true" or "something" then the  stillInvolved value will be set to "No"

Reply:
I've changed this to match what I've done elsewhere (in web and the SDK). Now specifically testing for `null` and `undefined`, which I think is better. See what you think.